client/coral-plugin-comment-count/CommentCount.js

```diff
@@ -3,8 +3,8 @@ import {I18n} from '../coral-framework';
 import translations from './translations.json';
 const name = 'coral-plugin-comment-count';
 
-const CommentCount = ({count}) => {
-  return <div className={`${name}-text`}>
+const CommentCount = ({count, onClick}) => {
+  return <div className={`${name}-text`} onClick={() => onClick()}>
```

Comment:
change the name to `handleClick` and pass it without wrapping it into another function if possible